doc/Command_Index_internal.tex

```diff
@@ -748,7 +740,7 @@ \section{Gregorio\TeX{} Controls}
 \end{argtable}
 
 \macroname{\textbackslash gre@char@cavum}{\#1\#2\#3\#4\#5\#6\#7\#8}{gregoriotex-signs.tex}
-Macro to typeset a "cavum" character.
+Macro to typeset a “cavum” character.
```

Comment:
OK.  I'll continue to use double apostrophes/backticks, but will tolerate unicode quotes.